core/objects_test.go

```diff
@@ -92,7 +106,7 @@ func TestChallengeSanityCheck(t *testing.T) {
 				AddressesResolved: []net.IP{net.IP{127, 0, 0, 1}},
 				AddressUsed:       net.IP{127, 0, 0, 1},
 			}}
-			test.Assert(t, chall.IsSane(false), "IsSane should be true")
+			test.Assert(t, chall.IsSane(true), "IsSane should be true")
```

Comment:
Clarifying comments from IRC

> rbarnes: The false checks are less stringent than the true checks.  The object was built to pass true, so it passes now, but since false < true it also passed before.
> jsha: I was misreading Assert(IsSane(true)) as Assert(IsSane() == true)
> jsha: the problems with boolean args
> jsha: so the test now says "treating this challenge as one that's been submitted for validation by the user, is it sane" where before it said "treating this challenge as one that was just now sent to the user, is it sane"
> jsha: so this change was not required to make the tests pass, right? this was a drive-by test cleanup?
> rbarnes: Right, drive-by